### ISSUE TITLE
Specify credentials for server shutdown

### ISSUE THREAD

**Issue by Billuc** [redhat-developer/vscode-server-connector#631]
Hello,

I was wondering if there was a way to specify the user and password to use during shutdown. I was using Eclipse previously and in my Wildfly configuration there was a Management Login Credentials section where I could specify them to be used during shutdown.

I kinda need to do this because if I don't I have an error sending the shutdown command and my server never stops.

Unfortunately, I have been going through the readme and the code and haven't found if it were possible or not :/ 

Can someone help me with this please ?

Thanks in advance !
Luc

**Comment by robstryker** [comment]
Unfortunately I've been looking over the code and don't see a spot where we allow customizing the shutdown arguments or including the management credentials in the shutdown process.  

I'll... see if there's anything that can be done here.

**Comment by Billuc** [comment]
I am working on a Tomcat server in parallel and there are configuration options like those : 

```json
  "args.shutdown.program.override.string": "stop",
  "args.shutdown.vm.override.string": "...",
```

Is it not possible to have something similar for Wildfly ? I could manage with that.
I am truly oblivious to the way the extension works so this may be a bad suggestion.

Thx

**Comment by robstryker** [comment]
Yes, this feature was added to the community servers / generic server framework, but was not added to the wildfly / eap ones. I will see when I can allocate time to get this added in, because it is a missing piece that really should belong here.

**Comment by robstryker** [comment]
Merged up in rsp-server.
